recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+docker_image:
+  - condaforge/linux-anvil-cuda:10.1
+  - condaforge/linux-anvil-cuda:10.0
+  - condaforge/linux-anvil-cuda:9.2
+CUDA_VER:
+  - 10.1
+  - 10.0
+  - 9.2
+zip_keys:
+  - - CUDA_VER
```

Comment:
Should there be two dashes here?

Reply:
Yeah the second dash means this is a group of things that should be zipped together, but it should not be zipped with anything else being zipped. For example we [zip `python` (and `vc` on Windows)]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1730aa4ed6f04bea8479e17e37de6598d92bb936/recipe/conda_build_config.yaml#L135-L139 ), which should be treated as a separate group.

Further conda-build requires that if we use nested lists, we must use them in all places. Otherwise it is an error.

ref: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#coupling-keys